Sources/Public/Consent+PublicAPI.swift

```diff
@@ -15,10 +15,17 @@ import Foundation
 
 @objc public extension Consent {
 
-    /// Retrieves the current consent preferences stored in the Consent extension.
-    /// - Parameter completion: invoked with the current consent preferences or an `AEPError` if an unexpected error occurs or the request timed out.
+    /// Retrieves the current consent preferences stored in the Consent extension
+    /// - Parameter completion: The completion handler is invoked with the current consent preferences and an optional error
+    /// - Parameter consents:  The current consent preferences as `[String: Any]`
+    /// - Parameter error:  An `AEPError`  if an unexpected error occurs or the request timed out
+    ///
+    /// Output example:
+    /// ```
+    /// ["consents": ["collect": ["val": "y"]]]
+    /// ```
```

Comment:
The params still don't show up 😞  
![Screen Shot 2021-04-01 at 5 21 27 PM](https://user-images.githubusercontent.com/33132425/113366670-bcb90780-930e-11eb-8cf7-faeb9f6d317b.png)

Reply:
That's because they are not parameters to the `getConsents` function... the only parameter this function takes is `completion`. The lines for `Parameter consents` and `Parameter error` need to be removed or consolidated into the `Parameter completion` line.